### DISCUSSION TITLE
French language finished

### DISCUSSION THREAD

**Discussion by jimmy49150** [aclap-dev/video-downloadhelper#1338] [Language & Translation]
Hi! I had fun finishing the French translation!
Good evening to all!

[fr.txt](https://github.com/user-attachments/files/17297719/fr.txt)

**Comment by Gluups** [comment]
A few more strings appeared after that, and after downloading I was asked to translate them.
I was hesitant about the distinction between recording (mémoriser) and saving, but thought it was no big deal and validated my translation.
I did not translate "Report", for I feared what in French we call "faux amis", words looking the same and having different meanings. Anyway, when seeing "report" I presume french people will understand, until this question is solved.

**Reply by Gluups** [reply]
in some contexts "report" can be "signaler"

**Comment by Gluups** [comment]
A further comment : the contextual display in the side panel did not work, this can lead to approximation in translations.

**Comment by Gluups** [comment]
To save my translations I had the idea to print to a PDF file. This saves only the beginning of each translation.
